configure-gcloud-docker-gcloud-v101/action.yml

```diff
@@ -19,10 +19,14 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: google-github-actions/auth@v0
+    - id: 'auth'
+      name: 'Authenticate to Google Cloud'
+      uses: 'google-github-actions/auth@v2'
       with:
         credentials_json: ${{ inputs.gcloud-token }}
-    - uses: google-github-actions/setup-gcloud@v1.0.1
+    - id: 'setup-gcloud'  # New step with your chosen ID
```

Comment:
Perfect, thanks for updating, one small leftover comment and we are good to go 🚀 

```diff
- # New step with your chosen ID
```

Reply:
ultra big face palm right here. been a long week.

Reply:
Happens to me all the time 😅